### DISCUSSION TITLE
Herkese Selamm

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#1478] [Soru Cevap - Türkçe] [answered]
Önemli bir gelişme varmı.  Hepimiz anlamsız bir şekilde 1 şubatımı bekliyoruz 🤪 

![Punch](https://user-images.githubusercontent.com/98268472/151692440-ff390830-2fa2-4e4a-8f20-d354637fbbae.jpg)

**Comment by ghost** [comment, marked as answer]
A.Selam kanka Henüz bişey yok.

**Reply by Skips1** [reply]
SAĞOL KANKA

**Reply by mustafasahints** [reply]
1 Şubatta ne olacak gençler?

**Reply by ghost** [reply]
Airdrop dağıtımı felan kanka bazı gelişmeler bekliyoruz açıkcası
